### ISSUE TITLE
WCAG: Contrast between 'delete' button text in RepeatingGroup row with errors

### ISSUE THREAD

**Issue by olemartinorg** [Altinn/app-frontend-react#2892]
### Description

When there's an error in a row, it will appear with a red background color:

![Image](https://github.com/user-attachments/assets/237e1992-f2ba-45ce-b971-0c4c5a3aebae)

According to [this comment](https://digdir.slack.com/archives/C076GU2QHCG/p1736414353174109?thread_ts=1736335017.237809&cid=C076GU2QHCG), we might get away with marking the row with the color `danger/surface-default` instead (#F5DCDC).

### Related

- #1664
